pkg/app/web/src/components/add-application-drawer/index.tsx

```diff
@@ -22,14 +22,15 @@ import {
 export interface AddApplicationDrawerProps {
   open: boolean;
   onClose: () => void;
+  onAdded: () => void;
```

Comment:
how about making this `onAdded` optional and make a default value for it on L33 👀 
In most of cases, looks like we only pass `onAdded={() => null}` for that 🤔

Reply:
Sure, that might be a good idea.
Thank you!